src/services/validateZendeskRequest.spec.ts

```diff
@@ -0,0 +1,412 @@
+import { validateZendeskRequest } from './validateZendeskRequest'
```

Comment:
Final suggestions:
> `validateZendeskRequest.ts` does not seem to validate rule 1 (check all fields are present) as per the ticket. Can we add that? 
> Check date values are not in the future.

Reply:
I've added a test to check the existence of all fields and we already have a tests in place for future dates.